src/Components/Search/Previews/Grids/ArtistSearch/MarketingCollections.tsx

```diff
@@ -107,13 +119,13 @@ export class MarketingCollectionsPreview extends React.Component<
 
         <Media lessThan="lg">
           <Flex alignItems="flex-start" flexWrap="wrap">
-            {items.slice(0, 3)}
+            {renderItems(this.props, 1)}
```

Comment:
I was able to move the `slice` into `renderItems`, where all decisions are made about how many things are in a row.